e2e/cypress/integration/messaging/inline_markdown_image_link_open_link_spec.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+describe('Messaging', () => {
+    let testTeam;
+    let testChannel;
+    let testUser;
+
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+        cy.apiInitSetup().then(({team, channel, user}) => {
+            testUser = user;
+            testTeam = team;
+            testChannel = channel;
+
+            cy.apiLogin(testUser);
+
+            // # Visit a test channel and post a message
+            cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+        });
+    });
+
+    it('MM-T188 - Inline markdown image that is a link, opens the link', () => {
+        // # Enable 'Show markdown preview option in message input box' setting in Account Settings > Advanced
+        cy.get('.sidebar-header-dropdown__icon').click();
+        cy.findByText('Account Settings').should('be.visible').click();
+        cy.get('#advancedButton').should('be.visible').click();
+        cy.get('#advancedPreviewFeaturesEdit').should('be.visible').click();
+        cy.get('#advancedPreviewFeaturesmarkdown_preview').should('be.visible').check();
+        cy.get('#saveSetting').should('be.visible').click();
+        cy.reload();
+
+        // # Post the provided Markdown text in the test channel
+        cy.focused().type('[![Build Status](https://travis-ci.org/mattermost/platform.svg?branch=master)](https://travis-ci.org/mattermost/platform)').type('{enter}');
```

Comment:
This is intermittently failing (missed typing first few characters into the textbox) as it seemed needed to wait some time. I suggest to use `cy.postMessage` instead.